Readme.rst

```diff
@@ -1,8 +1,18 @@
-.. image:: https://travis-ci.org/neos/neos-development-collection.svg
-    :target: https://travis-ci.org/neos/neos-development-collection
-.. image:: https://codeclimate.com/github/neos/neos-development-collection/badges/gpa.svg
+|Code Climate| |Travis Build Status| |Latest Stable Version| |Total Downloads| |Latest Unstable Version|
```

Comment:
Can we have the Travis build status as the first item? IMHO it's one of the most important ones.